packages/client/src/lib.js

```diff
@@ -70,18 +70,26 @@ class NFTStorage {
   /**
    * @param {API.Service} service
    * @param {Blob} blob
+   * @param {boolean} [isCar=false]
    * @returns {Promise<API.CIDString>}
    */
-  static async storeBlob({ endpoint, token }, blob) {
+  static async storeBlob({ endpoint, token }, blob, isCar = false) {
```

Comment:
Why not having a new function to `storeCar`?
Specially has the logic will increase for chunking, I feel like we should add a new function, rather than having users starting to use this function with a CAR and then need to change.
Also, as we will only support larger uploads with CAR files, we might reach a point that we want to deprecate (not recommend) storing Blobs?

Reply:
Yes, that's a good idea. I was going to create a storeCar function, as an alternative, but then got bogged down in figuring out all the various testing shims and typescript complaints. I went with the boolean flag initial as the original plan was to punt on having the client create CAR files itself, and I wanted to keep the change as small as possible... but boolean params are horrible `storeBlob(foobar, true)` is too mysterious.

Reply:
I think it's fine to add support for CAR in `storeBlob` - we can release this ASAP to enable client side CID generation and then when we have chunked uploads we can have a separate API and client method for `storeBigCar`. WDYT?

I agree the boolean param is horrible - can we just use the `type` property of the blob?

Reply:
i would rather just use the `type` param of the blob, but it would require the user to re-encode their file-blob to set the content type correctly if they are accepting `<input type='file'>`.

on discussion with @alanshaw we are going to go with @vasco-santos's suggestion here and change the client to add a `storeCar` method, which his work can then upgrade with large car support.